src/components/UpsertMovieModal.js

```diff
@@ -1,10 +1,29 @@
 import "./UpsertMovieModal.css";
 import React from "react";
-import { useForm } from "react-hook-form";
 import GenresRepo from "../services/GenreRepo";
 import { useDispatch } from "react-redux";
 import { bindActionCreators } from "redux";
 import { actionCreators } from "../store";
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+const MovieSchema = Yup.object().shape({
+    title: Yup.string().required(),
+    url: Yup.string().required(),
```

Comment:
It will be logically if it has url validation, because random string couldn't be a url. Yup has api for it

```suggestion
    url: Yup.string().url().required(),
```